packages/rest-api/src/routes/bridgeLimitsRoute.ts

```diff
@@ -0,0 +1,134 @@
+import { isTokenAddress } from '../utils/isTokenAddress'
+
+const router = express.Router()
+
+/**
+ * @openapi
+ * /bridgeLimits:
+ *   get:
+ *     summary: Get min/max origin values for bridge quote
+ *     description: Retrieve min/max bridgeable amounts to bridge from source chain to destination chain. Returns null for min/max amounts if limits are unavailable.
+ *     parameters:
+ *       - in: query
+ *         name: fromChain
+ *         required: true
+ *         schema:
+ *           type: integer
+ *         description: The source chain ID.
+ *       - in: query
+ *         name: toChain
+ *         required: true
+ *         schema:
+ *           type: integer
+ *         description: The destination chain ID.
+ *       - in: query
+ *         name: fromToken
+ *         required: true
+ *         schema:
+ *           type: string
+ *         description: The address of the token on the source chain.
+ *       - in: query
+ *         name: toToken
+ *         required: true
+ *         schema:
+ *           type: string
+ *         description: The address of the token on the destination chain.
+ *     responses:
+ *       200:
+ *         description: Successful response containing min and max origin amounts.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 maxOriginAmount:
+ *                   type: string
+ *                   description: Maximum amount of tokens that can be bridged from the origin chain.
+ *                 minOriginAmount:
+ *                   type: string
+ *                   description: Minimum amount of tokens that can be bridged from the origin chain.
+ *             example:
+ *               maxOriginAmount: "999600"
+ *               minOriginAmount: "4"
+ *       400:
+ *         description: Invalid input
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 error:
+ *                   type: object
+ *                   properties:
+ *                     value:
+ *                       type: string
+ *                     message:
+ *                       type: string
+ *                     field:
+ *                       type: string
+ *                     location:
+ *                       type: string
+ *             example:
+ *               error:
+ *                 value: "999"
+ *                 message: "Unsupported fromChain"
+ *                 field: "fromChain"
+ *                 location: "query"
+ *       500:
+ *         description: Server error
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 error:
+ *                   type: string
+ *                 details:
+ *                   type: string
+ */
+router.get(
+  '/',
```

Comment:
See other routes that have `normalizeNativeTokenAddress` and `checksumAddreses` middleware.

Reply:
Thanks, updated to use those middleware functions in https://github.com/synapsecns/sanguine/commit/30fd0c5fd08e6bfbc7ff9a2c71c09a87a9e4e634